docs/pbc/all/product-information-management/202212.0/install-and-upgrade/install-features/install-the-measurement-units-feature.md

```diff
@@ -362,6 +362,7 @@ Prepare your data according to your requirements using our demo data:
 
 ```yaml
 name,code,default_precision
+measurement_units.item.name,ITEM,1
```

Comment:
Entry with code `ITEM` is missing but used in other import files in the documentation.

Reply:
this unit is an infrastructural one and comes with install
see `\Spryker\Zed\ProductMeasurementUnit\ProductMeasurementUnitConfig::INFRASTRUCTURAL_MEASUREMENT_UNITS`